hybrasyl/Client.cs

```diff
@@ -629,7 +629,7 @@ public void Redirect(Redirect redirect, bool isLogoff = false)
             var x03 = new ServerPacket(0x03);
             x03.Write(addressBytes);
             x03.WriteUInt16((ushort)redirect.Destination.Port);
-            x03.WriteByte((byte)(redirect.EncryptionKey.Length + Encoding.GetEncoding(949).GetBytes(redirect.Name).Length + 7));
+            x03.WriteByte((byte)(redirect.EncryptionKey.Length + Encoding.ASCII.GetBytes(redirect.Name).Length + 7));

```

Comment:
Why remove this? This still works on 2.2?

Reply:
Code page 949 not found without extension which cause massive cpu usage issues.

Reply:
ah, yeah, I see that now `Unhandled Exception: System.NotSupportedException: No data is available for encoding 949. For information on defining a custom encoding, see the documentation for the Encoding.RegisterProvider method.` - are we sure this doesn't have any impact on the usda client?

Reply:
Yeah, I was not able to find any issues. And even after referencing some of Kyle's old data he utilized the ascii codepage

Reply:
👍